docs/personalization/triggers.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Triggers enable sending recommendations as emails to customers.
+---
+
+# E-mail triggers
+
+With e-mail triggers, you can increase the engagement of your visitors and customers by delivering recommendations straight to their mailboxes.
+As a store manager, you can expect that more engaged customers translate to bigger income, improved fulfillment of customer needs, while saving time and effort.
+
+## E-mail trigger types
+
+There are several e-mail triggers available in [[= product_name =]], including the following two:
+
+- Abandoned basket trigger: Personalization engine monitors the user's cart and pushes a message when cart status remains unchanged for a set time. The message contains items that have been abandoned in the cart.
+
+- Reactivation aka. "We miss you" trigger: Personalization engine monitors the user's overall activity and pushes a message when they have not returned to the site for a set time. Recommendations come from the [also purchased and also clicked](recommendation_models.md#also-clicked--purchased) models, while the [top purchased](recommendation_models.md#popularity-models) model is used as fallback.
```

Comment:
The entire sentence:
Recommendations come from the [also purchased and also clicked](recommendation_models.md#also-clicked--purchased) models, while the [top purchased](recommendation_models.md#popularity-models) model is used as fallback.

I would like to replace with:
Recommendations are adjusted to the user previous interests, based on the user activity. - something like that.